extensions/flac/src/main/java/com/google/android/exoplayer2/ext/flac/FlacDecoderJni.java

```diff
@@ -220,11 +221,12 @@ public long getNextFrameFirstSampleIndex() {
    * stream.
    *
    * @param timeUs A seek position in microseconds.
-   * @return The corresponding position (byte offset) in the flac stream or -1 if the stream doesn't
+   * @return The corresponding SeekPoints in the flac seek table or
+   * {@link com.google.android.exoplayer2.extractor.SeekPoint#position} set -1 if the stream doesn't
```

Comment:
It would probably be a lot clearer if this method returned `null` if the stream doesn't have a seek table?